src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write code here
+
+const list = document.querySelectorAll('li');
+
+function sortList(lists) {
+  return [...lists].sort((a, b) => {
+    const textA = a.dataset.salary.split('$').join('').split(',').join('');
+    const textB = b.dataset.salary.split('$').join('').split(',').join('');
+
+    return textB - textA;
+  }).forEach(el => el.parentNode.append(el));
+}
+
+function getEmployees(lists) {
+  const mass = [];
```

Comment:
what does mass means? 
```suggestion
  const result = [];
```